pyracing/response_objects/historical_data.py

```diff
@@ -94,84 +94,84 @@ def __init__(self, dict):
 
 class DriverStats:
     def __init__(self, dict):
-        self.avefieldsize = dict['13']
-        self.avefinishingposition = dict['9']
-        self.avestartingposition = dict['8']
-        self.avgincidents = dict['26']
-        self.avgpoints = dict['27']
-        self.clubid = dict['5']
-        self.clubname = parse_encode(dict['29'])
-        self.clubpoints = dict['21']
-        self.countrycode = parse_encode(dict['4'])
-        self.custid = dict['34']
-        self.displayname = parse_encode(dict['32'])
-        self.groupletter = parse_encode(dict['39'])
-        self.groupname = parse_encode(dict['24'])
-        self.helmcolor1 = parse_encode(dict['31'])
-        self.helmcolor2 = parse_encode(dict['33'])
-        self.helmcolor3 = parse_encode(dict['28'])
-        self.helmfacetype = dict['36']
-        self.helmhelmettype = dict['10']
-        self.helmpattern = dict['22']
+        self.avg_field_size = dict['13']
+        self.avg_finish_pos = dict['9']
+        self.avg_start_pos = dict['8']
+        self.avg_incidents = dict['26']
+        self.avg_points = dict['27']
+        self.club_id = dict['5']
+        self.club_name = parse_encode(dict['29'])
+        self.club_points = dict['21']
+        self.country_code = parse_encode(dict['4'])
+        self.cust_id = dict['34']
+        self.display_name = parse_encode(dict['32'])
+        self.group_letter = parse_encode(dict['39'])
+        self.group_name = parse_encode(dict['24'])
+        self.helm_color1 = parse_encode(dict['31'])
+        self.helm_color2 = parse_encode(dict['33'])
+        self.helm_color3 = parse_encode(dict['28'])
+        self.helm_face_type = dict['36']
+        self.helm_helmet_type = dict['10']
+        self.helm_pattern = dict['22']
         self.irating = dict['3']
         self.irating_rank = dict['2']
         self.laps = dict['7']
-        self.lapslead = dict['30']
-        self.licenseclass = parse_encode(dict['16'])
-        self.licenseclass_rank = dict['19']
-        self.licensegroup = dict['12']
-        self.licenselevel = dict['23']
+        self.laps_led = dict['30']
+        self.license_class = parse_encode(dict['16'])
+        self.license_class_rank = dict['19']
+        self.license_class = dict['12']
+        self.license_level = dict['23']
         self.points = dict['11']
         self.rank = dict['14']
         self.region = parse_encode(dict['38'])
-        self.rn = dict['37']
-        self.starts = dict['15']
-        self.sublevel = dict['35']
-        self.top25pcnt = dict['1']
+        self.row = dict['37']
+        self.starts_total = dict['15']
```

Comment:
I think starts here is fine, but if we want to include the word total, I would say it should be `total_starts`